arbnode/espresso_caff_node.go

```diff
@@ -239,6 +234,7 @@ func (n *EspressoCaffNode) createBlock(ctx context.Context) (returnValue bool) {
 
 func (n *EspressoCaffNode) Start(ctx context.Context) error {
 	n.StopWaiter.Start(ctx, n)
+	n.espressoStreamer.Start(ctx)
```

Comment:
Previously espressoStreamer is lazy. Now we ask it to start a go routine to fetch blocks.